hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/OperationCallbacks.java

```diff
@@ -119,7 +119,8 @@ void deleteObjectAtPath(Path path,
       Path path,
       S3AFileStatus status,
       boolean collectTombstones,
-      boolean includeSelf) throws IOException;
+      boolean includeSelf,
+      boolean rejectAuthoritative) throws IOException;
```

Comment:
You are right. At first I introduced the extra parameter because I thought some tools might use the improvement in this PR and other can continue to work the same way by forcing non-authoritative. Given that only Import tool use this method and it should force non-authoritative, implementers of this method should force non-authoritative. The flag has no use.

Reply:
Thanks!